### PR TITLE
Fix/pagination

### PR DESCRIPTION
**Pull request by billoosijok**
* Adds pagination to contributions table
* Implements sorting using the default MUI column sorting

### CODE REVIEW

File: app/src/Pages/Portal/Contributions/ContributionsTable/TablePagination.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+          SelectProps={{
+            inputProps: { 'aria-label': 'rows per page' },
+            native: true,
+          }}
+          {...rest}
+        />
+      </div>
+    );
+  }
+  return (
```
Comment (andrewbiang888) [inline app/src/Pages/Portal/Contributions/ContributionsTable/TablePagination.js:23]: Delete this if it's not being used

File: app/src/Pages/Portal/Contributions/index.js [hunk 2]
```diff
@@ -27,14 +27,10 @@ class ContributionsPage extends React.Component {
...
+      status: filterOptions.status,
     };
-    if (filterOptions.field || filterOptions.direction) {
-      data.sort = {
-        field: filterOptions.field,
-        direction: filterOptions.direction,
-      };
-    }
 
+    console.log(data);
```
Comment (andrewbiang888) [inline app/src/Pages/Portal/Contributions/index.js]:
```suggestion
   
```

File: app/src/components/Forms/FilterContributions/FilterContributions.js [hunk 3]
```diff
@@ -56,39 +56,39 @@ const fields = {
...
-  perPage: {
-    label: 'Results per page',
-    section: 'paginate',
-    options: {
-      values: ['50', '100', '150'],
-    },
-    component: SelectField,
-    validation: Yup.string('Choose number of results per page'),
-  },
+  // orderBy: {
```
Comment (andrewbiang888) [inline app/src/components/Forms/FilterContributions/FilterContributions.js]: delete this too if it's not being used

**Review by andrewbiang888 — CHANGES_REQUESTED** [review]
Clean up a few things then looks good to me!

File: app/src/Pages/Portal/Contributions/ContributionsTable/TablePagination.js (hunk 1 above)
Comment (billoosijok) [reply app/src/Pages/Portal/Contributions/ContributionsTable/TablePagination.js:23]: Will do

**Review by andrewbiang888 — APPROVED** [review]
Nice job

**Comment by kdv24** [comment]
Nice work! It looks great.

Looks like there may still be a bit of an issue on pagination though; if I set it to 100 or 150, it shows screwiness like-  `101-70 of 70`.
Screen Shot 2019-09-09 at 5 10 34 PM

**Pull request merged**